memcached_prefix

```diff
@@ -1 +0,0 @@
-689f5ba0a0bab644aaa29626c326d455:
```

Comment:
oh, I see, you fixed memcached_prefix having been committed, but in the second commit, so it gets added and then removed in this branch